roles/image_builder/README.md

```diff
@@ -0,0 +1,113 @@
+Role Name
+=========
+
+This role automates the setup server and [osbuild](https://www.osbuild.org/) [compose builds](https://www.osbuild.org/guides/user-guide/user-guide.html) using the osbuild backend [Weldr](https://weldr.io/) API with Microshift and necessary dependencies installed.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+### builder_blueprint_name
```

Comment:
docs don't appear to match the variable names used below

Reply:
One day, one, day. I will complete this PR :laughing: 

Give me one sec :smile: